README.md

```diff
@@ -88,21 +108,31 @@ from confluent_kafka import KafkaError
 from confluent_kafka.avro import AvroConsumer
 from confluent_kafka.avro.serializer import SerializerError
 
-c = AvroConsumer({'bootstrap.servers': 'mybroker,mybroker2', 'group.id': 'groupid', 'schema.registry.url': 'http://127.0.0.1:8081'})
+
+c = AvroConsumer({
+    'bootstrap.servers': 'mybroker,mybroker2',
+    'group.id': 'groupid',
+    'schema.registry.url': 'http://127.0.0.1:8081'})
+
 c.subscribe(['my_topic'])
-running = True
-while running:
+
+while True:
     try:
         msg = c.poll(10)
-        if msg:
-            if not msg.error():
-                print(msg.value())
-            elif msg.error().code() != KafkaError._PARTITION_EOF:
-                print(msg.error())
-                running = False
+
     except SerializerError as e:
-        print("Message deserialization failed for %s: %s" % (msg, e))
-        running = False
+        print("Message deserialization failed for {}: {}".format(msg, e))
+        break
+
+    if msg:
```

Comment:
I see that the original code has a bug:
Bool comparison operator calls _msg.__len__, which is implemented to return the number of bytes of the message **value**. Tombstone messages however have a null value (but a non-empty key) and thus a __len__ of 0, which will make these messages ignored by this code.

So this should be changed to:
```
   if msg is None:
       continue
   if msg.error():
...
```